packages/beeq/src/components/tab-group/bq-tab-group.tsx

```diff
@@ -117,10 +129,18 @@ export class BqTabGroup {
     if (!isHTMLElement(target, 'bq-tab')) return;
 
     switch (event.detail.key) {
+      case 'ArrowDown': {
+        await this.focusTabSibling(target, 'forward');
+        break;
+      }
       case 'ArrowRight': {
         await this.focusTabSibling(target, 'forward');
         break;
       }
+      case 'ArrowUp': {
+        await this.focusTabSibling(target, 'backward');
+        break;
+      }
```

Comment:
Perhaps we could use a lookup object instead of a switch statement, which is a bit more simple, efficient, and easy to understand. It avoids the verbosity of ithe switch statements while providing clear mappings between keys and actions.

```js
@Listen('bqKeyDown', { passive: true })
async onBqKeyDown(event: CustomEvent<KeyboardEvent>) {
  const { target } = event;

  // Ensure the target is an HTML element with the tag name 'bq-tab'
  if (!isHTMLElement(target, 'bq-tab')) return;

  // Map of key actions to their corresponding directions
  const keyActions: { [key: string]: 'forward' | 'backward' } = {
    ArrowDown: 'forward',
    ArrowRight: 'forward',
    ArrowUp: 'backward',
    ArrowLeft: 'backward',
  };

  // Get the direction based on the pressed key
  const direction = keyActions[event.detail.key];

  // If a valid direction is found, focus the sibling tab
  if (direction) {
    await this.focusTabSibling(target, direction);
  }
}
```